tfx_addons/example_filter/filter_function.py

```diff
@@ -0,0 +1,44 @@
+"""Filters the data from input data by using the filter function.
+
+Args:
+  x_list: Input list of data to be filtered.
+
+
+Returns:
+  filtered list
+
+"""
+
+
+# TODO: This license is not consistent with the license used in the project.
```

Comment:
swap license with doc string - pls also see my comment above regarding the module doc string